### PR TITLE
feat: add Prettier config #P23-162

### PR DESCRIPTION
**Pull request by Vectari**
Here is my proposition of Prettier configuration. If you have any suggestion about rules (.prettierrc) or ignore (.prettierignore) just let me know.
Make sure that you have checked settings in VS Code:

1. Editor: Format On Save
2. Editor: Default Formatter -> Prettier - Code formatter

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "semi": true,
+  "tabWidth": 2,
+  "singleQuote": true,
```
Comment (aniasgit) [inline .prettierrc:4]: Do we want a `singleQuote`? Until now, double quote was used almost everywhere.

Comment (Vectari) [reply .prettierrc:4]: I use double too but that's how I understood Wojtek's suggestions. Maybe I understand it wrong...

Comment (aniasgit) [reply .prettierrc:4]: Look here https://github.com/intive/patronage2023-js/pull/19#discussion_r1152118840

Comment (Vectari) [reply .prettierrc:4]: Yes, I seen it. Maybe I understand something wrong. I fix it after work.

Comment (Vectari) [reply .prettierrc]: On phone I can too 🙂

**Review by wlechowicz — APPROVED** [review]

**Review by wyola — APPROVED** [review]
looks good to me

**Review by aniasgit — APPROVED** [review]

**Pull request merged**